ui/app/components/app/gas-customization/gas-modal-page-container/advanced-tab-content/advanced-tab-content.component.js

```diff
@@ -18,6 +18,7 @@ export default class AdvancedTabContent extends Component {
     isSpeedUp: PropTypes.bool,
     customGasLimitMessage: PropTypes.string,
     minimumGasLimit: PropTypes.number,
+    customPriceIsExcessive: PropTypes.bool,
```

Comment:
Nit: Could we make this required? If I understand this correctly, it should always be passed in

```suggestion
    customPriceIsExcessive: PropTypes.bool.isRequired,
```

Reply:
I think the same applies for most of the props added in this PR

Reply:
Yep should be required, I updated others as well